packages/react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecText.java

```diff
@@ -580,6 +595,7 @@ public void setActiveFormats(Iterable<String> newFormats) {
                     break;
                 case "underline":
                     newFormatsSet.add(AztecTextFormat.FORMAT_UNDERLINE);
+                    break;
```

Comment:
I interpret this adds an overlooked `break` that should've always been here. Is that true or does this change serve a specific purpose?

Reply:
That's correct, looking back when I added this in https://github.com/WordPress/gutenberg/pull/36028 I missed adding the `break` when adding the `mark` case.